src/sentry/api/urls.py

```diff
@@ -1967,6 +1968,11 @@
         ProjectReleaseFileDetailsEndpoint.as_view(),
         name="sentry-api-0-project-release-file-details",
     ),
+    url(
+        r"^(?P<organization_slug>[^\/]+)/(?P<project_slug>[^\/]+)/artifact_lookup/$",
+        ArtifactLookupEndpoint.as_view(),
```

Comment:
It should be prefixed:

```suggestion
        ProjectArtifactLookupEndpoint.as_view(),
```

I also wonder if we want to have `OrganizationArtifactLookupEndpoint`, or do we not want to support org-wide endpoints in this case?

Reply:
Right now it does not use any project-specific APIs inside. I believe there is one lookup that needs to happen on a per-project basis, though I’m not entirely sure. Either way, this is very much specific to the symbolicator usecase, this is not intended to be public at all.